pyaqueduct/experiment.py

```diff
@@ -35,10 +35,16 @@ class Experiment(BaseModel):
     created_at: datetime
     "Creation datetime of the experiment."
 
+    def __init__(
+        self, experiment_id: UUID, alias: str, created_at: datetime, client: AqueductClient, **data
+    ):
+        super().__init__(id=experiment_id, alias=alias, created_at=created_at, **data)
```

Comment:
why should the variable change its name from child (experiment_id) class to base class (id)? Is this because of database schema?